### PR TITLE
Manage subscription page

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Manage subscription page

Add page, routing, update link
Add service and factory functions

Create filter for subscription description

[stage-2]

## Motivation and Context
Billing Frictions epic

## How Has This Been Tested?
Tested changes locally. This is going to be the base PR for the functionality.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc @santiagonoguez fyi base PR for this. Thanks.

File: web/partials/billing/subscription.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  </div>
+
+  <div rv-spinner rv-spinner-key="subscription-loader" rv-spinner-start-active="1">
+
+  <div id="errorBox" ng-show="billingFactory.apiError" class="madero-style alert alert-danger" role="alert">
+    <strong>{{billingFactory.apiError}}</strong>
+  </div>
+
+  <h4>
+    {{billingFactory.subscription.subscription | subscriptionDescription}} —
```
Comment (santiagonoguez) [inline web/partials/billing/subscription.html:14]: subscription.subscription :O :O :O

Comment (santiagonoguez) [reply web/partials/billing/subscription.html:14]: :D

Comment (alex-deaconu) [reply web/partials/billing/subscription.html:14]: @santiagonoguez it's not a typo! I will probably rename the factory object to reflect that it has a bunch of information in it. Thanks.

**Comment by alex-deaconu** [comment]
@ezequielc @santiagonoguez I believe this is now ready for the final review before we can release. Thanks!

**Comment by ezequielc** [comment]
@alex-deaconu I reviewed and in terms of functionality it looks good. But I think we need to improve the mobile UI, adding margins between boxes and stretching them to fit horizontally. Thanks!

**Comment by alex-deaconu** [comment]
@ezequielc sounds good, I will take a look. Thanks!

**Review by ezequielc — APPROVED** [review]
@alex-deaconu  Thanks for addressing the layout. Looks good 👍

**Pull request merged**